hepmc.sh

```diff
@@ -1,6 +1,6 @@
 package: HepMC
 version: "%(tag_basename)s"
-tag: v2.06.10
+tag: 2.06.10
 source: https://gitlab.cern.ch/hepmc/HepMC
```

Comment:
The source has to be modified 
- from https://gitlab.cern.ch/hepmc/HepMC 
- to     https://gitlab.cern.ch/hepmc/HepMC.git

Note the ".git" missing at the end ; that is what prevents the HepMC sources to be downloaded apparently...
I have figured it out by looking at the alidist recipes provided for 'private' sources distributed via gitlab that we have in alisw, like epos.sh and dmpjet.sh.
This seems to cure the issue : it compiles under my alidock build at least.